.github/workflows/all-post-commit-workflows.yaml

```diff
@@ -11,11 +11,16 @@ permissions:
   contents: read
   pages: write
   id-token: write
+  packages: write
 
 jobs:
   static-checks:
     uses: ./.github/workflows/all-static-checks.yaml
     secrets: inherit
+  build-wheels:
+    needs: build-artifact
```

Comment:
Haha great renaming

Wonder if we can eventually merge wheels and binary artifacts jobs? @TT-billteng

Reply:
you mean build artifact workflow would create both?

Reply:
I do not see why not. But not this PR :)

Reply:
true that can work :D

Reply:
only disadvantage would be that C++ tests could only start after wheel is built, which it wouldn't need